src/assets/question-mark-icon.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   id="svg60"
+   version="1.1"
+   width="24"
+   viewBox="0 0 24 24"
+   height="24">
+  <defs
+     id="defs64" />
+  <path
+     id="path58"
+     d="m 11,18 h 2 V 16 H 11 Z M 12,6 C 9.79,6 8,7.79 8,10 h 2 c 0,-1.1 0.9,-2 2,-2 1.1,0 2,0.9 2,2 0,2 -3,1.75 -3,5 h 2 c 0,-2.25 3,-2.5 3,-5 0,-2.21 -1.79,-4 -4,-4 z" />
+</svg>
```

Comment:
Where did you get this SVG?

Reply:
Nevermind, I found where you got it. Can't believe the standard material icon font doesn't have a question mark.